server/lib/flow-event.js

```diff
@@ -23,7 +24,98 @@ var HOSTNAME = os.hostname();
 var MAX_DATA_LENGTH = 100;
 var VERSION = 1;
 
-module.exports = function (event, data, request) {
+const FLOW_BEGIN_EVENT_TYPES = /^flow\.[a-z_-]+\.begin$/;
```

Comment:
I leave it up to you. There has been a lot of back and forth as a group about how/when to make ES6 updates. Some prefer to do it as part of other PRs, I, like you, generally prefer to do them standalone.